recipes/r-immucellaimouse/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "1.0" %}
+
+package:
+  name: r-immucellaimouse
+  version: {{ version }}
+
+source:
+  url:
+    https://github.com/lydiaMyr/ImmuCellAI-mouse/archive/refs/heads/main.zip
+  sha256: f83fb931ca2dc401d8d569f96620c709613934e4817ef12193b13f5cdf60d154
+
+build:
+  run_exports:
+    - {{ pin_subpackage('r-immucellaimouse', max_pin="2") }}
```

Comment:
What does this do? I've never seen this before...

Reply:
I don't know. I tried submitting a recipe the usual way and it complained because this section was missing (check also the guidelines just above)

Reply:
But apparently adding that section makes it work. 
However now the package is not found when called with `library()`